### PR TITLE
Removed the generator and db related stuff 

### PR DESCRIPTION
**Pull request by lukasz-gryzbon**

### CODE REVIEW

File: src/integrationTest/resources/application-test.properties [hunk 1]
```diff
@@ -1,5 +1,7 @@
 lsd.dist.db.connectionString=mongodb://localhost:27017
+#lsd.dist.db.connectionString=mongodb://root:mongo@localhost:27017
```
Comment (nickmcdowall) [inline src/integrationTest/resources/application-test.properties:2]: might have missed this commented out line?

File: src/main/java/io/lsdconsulting/lsd/distributed/interceptor/config/ObfuscatorPropertyConfig.java [hunk 2]
```diff
@@ -11,8 +10,8 @@
...
 @Setter
 @Configuration
-@ConfigurationProperties(prefix = "lsd.dist.obfuscator")
+//@ConfigurationProperties(prefix = "lsd.dist.obfuscator")
 public class ObfuscatorPropertyConfig {
 
-    private String sensitiveHeaders;
-}
\ No newline at end of file
+//    private String sensitiveHeaders;
```
Comment (nickmcdowall) [inline src/main/java/io/lsdconsulting/lsd/distributed/interceptor/config/ObfuscatorPropertyConfig.java]: couple comments missed here too?

**Review by nickmcdowall — APPROVED** [review]
👍  nicely cleaned up (couple commented out lines might have been missed)

File: src/integrationTest/resources/application-test.properties (hunk 1 above)
Comment (lukasz-gryzbon) [reply src/integrationTest/resources/application-test.properties:2]: thanks!

File: src/main/java/io/lsdconsulting/lsd/distributed/interceptor/config/ObfuscatorPropertyConfig.java (hunk 2 above)
Comment (lukasz-gryzbon) [reply src/main/java/io/lsdconsulting/lsd/distributed/interceptor/config/ObfuscatorPropertyConfig.java]: Thanks!

**Pull request closed without merging**